pages/history.js

```diff
@@ -1,10 +1,62 @@
 import Head from "next/head";
 import styled from "styled-components";
 
+const Body = styled.p`
+	font-size: 1.5rem;
+	text-align: center;
+`;
+
 export default () => (
 	<>
 		<Head>
 			<title>History - The Snakeroom</title>
 		</Head>
+		<Body>
+			<strong>April 2020 - Imposter</strong>
+			<br />
+			Imposter is a Turing Test, the player receives four messages from
```

Comment:
```suggestion
			Imposter was a Turing test where the player received four messages from
```

Reply:
This entire article is written in present tense as would be expected for a review. A review is conducted with the implication that the quoted source material is timeless and therefore referred to in the present tense regardless of when the review is written or read

Reply:
[Wikipedia disagrees.](https://en.wikipedia.org/wiki/Place_(Reddit))

Reply:
I am not concerned in which tense Wikipedia references Place.

Reply:
The replacement of a comma with the word where has been applied, the change in tense has not.